rules/rules.libsonnet

```diff
@@ -15,6 +15,17 @@
           },
         ],
       },
+      {
+        name: 'mintel-disk.rules',
+        rules: [
+          {
+            record: 'mintel:pvc:inodes_free:percentage',
+            expr: |||
+              (kubelet_volume_stats_inodes_free / kubelet_volume_stats_inodes) * 100
```

Comment:
please add the "Kubelet Selector" using the variable, similar the other rule in this file

Reply:
Done